circle.yml

```diff
@@ -0,0 +1,68 @@
+machine:
+  environment:
+    # We need to set this variable to let Anaconda take precedence
+    PATH: "/home/ubuntu/miniconda/envs/circleenv/bin:/home/ubuntu/miniconda/bin:$PATH"
+    MNE_DATA: "/home/ubuntu/mne_data"
```

Comment:
And you shouldn't actually need to set this because `utils` should automatically try to use this location, right?